src/ServiceControl/Infrastructure/RavenDB/Expiration/ErrorMessageCleaner.cs

```diff
@@ -88,6 +104,22 @@ public static void Clean(int deletionBatchSize, DocumentDatabase database, DateT
                 return results.Count(x => x.Deleted == true);
             }, items, database, token);
 
+            var deletedFailedMessageRetry = Chunker.ExecuteInChunks(failedRetryItems.Count, (itemsForBatch, db, s, e) =>
```

Comment:
Do we want to include this count in the log message below?

Should we consider putting this (and the attachment deletion) before the message deletion? That would prevent orphaned docs if we clean up the message and then crash.

Reply:
Good catch!